src/sentry/models/artifactbundle.py

```diff
@@ -73,6 +77,13 @@ def get_release_dist_pair(
         except IndexError:
             return None, None
 
+    @classmethod
+    def delete_old_entries(cls, expired_threshold: Optional[int]):
+        expiration_time = timezone.now() - timedelta(
+            days=expired_threshold or OLD_ENTRIES_THRESHOLD
+        )
+        ArtifactBundle.objects.filter(date_added__lt=expiration_time).delete()
```

Comment:
Are you sure this will trigger the deletions as configured in https://github.com/getsentry/sentry/pull/46366 ?
I think you will have to use this code path ~ https://github.com/getsentry/sentry/blob/master/src/sentry/runner/commands/cleanup.py#L192-L195